src/services/backingImage.js

```diff
@@ -28,9 +28,21 @@ export async function deleteBackingImage(params) {
   }
 }
 
+export async function bulkDownload(images) {
+  for (const image of images) {
+    if (image?.name) {
+      // eslint-disable-next-line no-underscore-dangle
+      const downloadUrl = `${window.__pathname_prefix__}${window.__pathname_prefix__.endsWith('/') ? '' : '/'}v1/backingimages/${image.name}/download`
+      // specific way to download multiple files with one click, User may allow the browser to download multiple files at once
+      Object.assign(document.createElement('a'), { href: downloadUrl, download: image.name }).click()
```

Comment:
Current backing image [download API](https://github.com/a110605/longhorn-ui/blob/2021fa07e0b88e4ee36debb1badbd01ef28fa31d/src/services/backingImage.js#L42) only support download one image at a time
```
v1/backingimages/${params.name}/download
```
This workaround is to create multiple download link and clikc, but need user to give browser permission to do that. So I add a note in modal to let user know
<img width="1496" alt="Screenshot 2024-05-18 at 4 14 46 PM" src="https://github.com/longhorn/longhorn-ui/assets/5744158/20a83f4a-b6f4-4d02-8257-5deaeda409b8">
<img width="498" alt="Screenshot 2024-05-20 at 10 20 41 AM" src="https://github.com/longhorn/longhorn-ui/assets/5744158/73e7cb35-d6e9-4056-ae0b-5fdfd6a4f069">




The best approach is to zip the selected image from backend and has a new API to download that zip file. This is also how dropbox and google drive support download multiple files.


Refer :  https://stackoverflow.com/questions/52072098/how-to-download-multiple-files-with-window-open#comment91093930_52072098:~:text=Better-,to,-return%20a%20ZIP